wiki/Welcome/en.md

```diff
@@ -17,9 +17,9 @@ Welcome to _osu!_, a free-to-win rhythm game developed by peppy with four game m
 
 ## Article of the month
 
-_See also: [Ranking Criteria](/wiki/Ranking_Criteria)._
+_See also: [Performance Points](/wiki/Performance_Points)._
 
-Ranking Criteria is a set of rules and guidelines set by the QAT to ensure that every player can expect similar game play throughout every beatmap.
+Performance Points is a ranking metric that aims to be more contextually relevant to a player's progression in a continuous game like osu!. It tries to do this by shifting the focus of skill progression from the amount of time played to the actual representation of the player's skill.
```

Comment:
something about this reads really off to me, should be something like

``Performance Points are a value from a ranking metric that aims to be more contextually relevant..`` etc